app/lib/vita_partner_importer.rb

```diff
@@ -8,10 +8,8 @@ def self.upsert_vita_partners(yml = VITA_PARTNERS_YAML)
 
     yaml_partners = YAML.load_file(yml)['vita_partners']
     yaml_zendesk_group_ids = yaml_partners.map { |partner| partner["zendesk_group_id"] }
-    VitaPartner.transaction do
```

Comment:
I removed the `VitaPartner.transaction do` from this method because it's simpler conceptually to me if the Rake task is the one managing transactions.

I don't think this change is necessary, but I like it. Happy to undo it if desired.

Reply:
If we move the logic to be part of the rake task, when this runs in tests or someone were to run it in rails c, it wouldn't roll back if it fails somewhere. are we okay with that tradeoff?

Reply:
Given that each test runs in a transaction per https://relishapp.com/rspec/rspec-rails/docs/transactions (we have `config.use_transactional_fixtures = true`), it will automatically roll back if it fails in a test.

If someone runs it in local development `rails c`, then they could have a problem. I'm not very sympathetic to that.

If someone runs it in prod `rails c`, then we could have a problem. Good eye. However, we have a safe way to run it (`rake db:upsert_vita_partners`), and that's how Aptible runs it (per our `.aptible.yml`). Given that, I'm OK with this change.

Let me know if you think a follow-up commit is important! I'll roll forward with this for now.